cmd/snap/cmd_find.go

```diff
@@ -65,9 +60,29 @@ func getPrice(prices map[string]float64, currency, status string) string {
 		}
 	}
 
+	return val, currency, nil
+}
+
+func formatPrice(val float64, currency string) string {
 	return fmt.Sprintf("%.2f%s", val, currency)
 }
 
+func getPriceString(prices map[string]float64, suggestedCurrency, status string) string {
+	price, currency, err := getPrice(prices, suggestedCurrency)
+
+	// If there are no prices, then the snap is free
+	if err != nil {
+		return ""
+	}
+
+	// If the snap is priced, but has been purchased
+	if status == "available" {
```

Comment:
I think == "available" is clearer, so will change the other instance to be the same